azafea/tests/integration/migrations/test_migrations.py

```diff
@@ -7,26 +7,19 @@
 # file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
 
-import os
-
-import pytest
+from importlib import import_module
+from pathlib import Path
 
 from .. import IntegrationTest
 
+TEST_DIR = Path(__file__).parent
 
-class TestEmptyMigrations(IntegrationTest):
-    # This needs to run before the "setup_teardown" method, and it seems pytest runs fixtures in
-    # alphabetical order when they have the same scope, which is what makes this work.
-    # FIXME: Can we control the order explicitly to be safer?
-    @pytest.fixture(autouse=True)
-    def set_handler_module(self, handler_with_migrations):
-        self.handler_module = handler_with_migrations.basename
-        self._handler_path = handler_with_migrations
 
+class TestEmptyMigrations(IntegrationTest):
     def test_make_empty_migration(self):
         self.run_subcommand('make-migration', 'test_make_empty_migration')
 
-        migrations = (self._handler_path / 'migrations').listdir()
```

Comment:
AFAICT `pathlib.Path` never had a `.listdir()` method – how did this work before?!

Reply:
Because it used to be a [tmpdir fixture](https://docs.pytest.org/en/6.2.x/tmpdir.html#the-tmpdir-fixture), which returns a `py.path.local` object. I've never looked closely, but I think it's basically a precursor to `pathlib.Path`.